css/style.css

```diff
@@ -0,0 +1,201 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body{
+    font-size: 15px;
+    font-family: 'Open Sans', sans-serif;
+    color: #FFFFFF;
+}
+
+.gutter{
+    width:80%;
+    margin: 0 auto;
+}
+
+.split1{
+    background-color: #34ADFF;
```

Comment:
Is background colour needed?

Reply:
Webkit not support in IE10 so works as a fallback colour